edb/lib/cal.edgeql

```diff
@@ -266,14 +318,51 @@ cal::date_get(dt: cal::local_date, el: std::str) -> std::float64
                 detail => (
                     '{"hint":"Supported units: century, day, ' ||
                     'decade, dow, doy, isodow, isoyear, ' ||
-                    'millenium, month, quarter, seconds, week, year."}'
+                    'millennium, month, quarter, seconds, week, year."}'
                 )
             )
         END
     $$;
 };
 
 
+CREATE FUNCTION
+cal::duration_normalize_hours(dur: cal::relative_duration)
+  -> cal::relative_duration
+{
+    CREATE ANNOTATION std::description :=
+        'Convert 24-hour chunks into days.';
+    SET volatility := 'Immutable';
+    SET force_return_cast := true;
+    USING SQL FUNCTION 'justify_hours';
+};
+
+
+CREATE FUNCTION
+cal::duration_normalize_days(dur: cal::relative_duration)
+  -> cal::relative_duration
+{
+    CREATE ANNOTATION std::description :=
+        'Convert 30-day chunks into months.';
+    SET volatility := 'Immutable';
+    SET force_return_cast := true;
+    USING SQL FUNCTION 'justify_days';
+};
+
+
+CREATE FUNCTION
+cal::duration_normalize(dur: cal::relative_duration)
```

Comment:
Can you normalize date_duration?

Reply:
Added an overload for normalizing `date_duration` returning `date_duration` as the result.

Reply:
Yes, and I also removed the convenience `duraiton_normalize` function that combines 2 other normalizations.